geomstats/geometry/group_action.py

```diff
@@ -29,33 +30,37 @@ def __call__(self, group_elem, point):
         """
 
 
-class LieAlgebraBasedGroupAction(GroupAction):
+class ComposeAction(GroupAction):
```

Comment:
Thanks! It makes it easier to read.

But now it makes me wonder: why do we need this class in addition to the already existing `Group` (or `LieGroup` or `MatrixLieGroup`?) class?

I think we should rename `ComposeAction` to `Group` (or `LieGroup` or `MatrixLieGroup`?) and import here the code from `Group` (or `LieGroup` or `MatrixLieGroup`?).

A group is basically a set with a group action acting on itself...

Reply:
Yes, I agree that GroupAction is different than Group.

Yes, our current ComposeGroupAction is, in effeect,  a group. A group is G x G -> G